source/common/router/config_impl.h

```diff
@@ -457,6 +457,56 @@ class RouteTracingImpl : public RouteTracing {
   Tracing::CustomTagMap custom_tags_;
 };
 
+/**
+ * Implementation of PathMatchPolicy that reads from the proto
+ * PathMatchPolicy of the RouteAction.
+ */
+class PathMatchPolicyImpl : public PathMatchPolicy {
+public:
+  PathMatchPolicyImpl(const envoy::config::core::v3::TypedExtensionConfig typed_config,
+                      ProtobufMessage::ValidationVisitor& validator, std::string path);
+
+  // Default constructor that disables internal redirect.
+  PathMatchPolicyImpl();
+
+  bool enabled() const override { return enabled_; }
+
+  PathMatchPredicateSharedPtr predicate() const override;
+
+  std::string path() const { return path_; }
+
+private:
+  const std::string path_;
```

Comment:
Similarly, I see where this gets initialized, but not any uses other than the accessor, which itself seems unused.

Reply:
Removed it